templates/plugin/ansible.env.erb

```diff
@@ -0,0 +1,13 @@
+ANSIBLE_CALLBACK_WHITELIST="<%= @callback %>"
```

Comment:
the `_WHITELIST` name is deprecated since 2.11 and will be removed in 2.15. the new name is `ANSIBLE_CALLBACKS_ENABLED` - to be super sure we could set both for now?

Reply:
Done, but ansible still complains :/

Reply:
Yeah, it will complain until we don't set the `WHITELIST`.

We could drop that, but it would mean we're not compatible with the Ansible as shipped in Debian stable.